tasks-by-users/AbbasShaikh0304/01-E-Commerce website/cart.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/fontawesome.min.css"
+    />
+
+    <link rel="stylesheet" href="Style.css" />
+    <title>Cart</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="navbar">
+        <div class="logo">
+          <a href="index.html"
+            ><img src="images/logo.png" alt="RedStore" width="125px"
+          /></a>
+        </div>
+        <nav>
+          <ul id="MenuItems">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="products.html">Products</a></li>
+            <li><a href="">About</a></li>
+            <li><a href="">Contacts</a></li>
+            <li><a href="account.html">Account</a></li>
+          </ul>
+        </nav>
+        <a href="cart.html"
+          ><img src="images/cart.png" alt="Cart" width="30px" height="30px"
+        /></a>
+        <img src="images/menu.png" class="menu-icon" onclick="menutoggle()" />
+      </div>
+    </div>
+
+    <!-------------Cart item Details---------------->
+
+    <div class="small-container cart-page">
+      <table>
+        <tr>
+          <th>Product Name</th>
+          <th>Quantity</th>
+          <th>Total</th>
+        </tr>
+        <tr>
+          <td>
+            <div class="cart-info">
+              <a href="Products-details.html"
+                ><img src="images/buy-1.jpg" alt="Red Printed T-Shirt"
+              /></a>
+              <div>
+                <a href="Products-details.html"><p>Red Printed T-shirt</p></a>
+                <small>Price:$50.00</small> <br />
+                <a href="">Remove</a>
+              </div>
+            </div>
+          </td>
+          <td><input type="number" value="1" /></td>
+          <td>$50.00</td>
+        </tr>
+        <tr>
+          <td>
+            <div class="cart-info">
+              <img src="images/buy-2.jpg" alt="Black Sneaker" />
+              <div>
+                <p>Black Sneaker Shoes</p>
+                <small>Price:$65.00</small> <br />
+                <a href="">Remove</a>
```

Comment:
```suggestion
                <a href="#">Remove</a>
```

Reply:
now
Thankyou for you suggestions. Commited the Changes @coder-mezab